test/extensions/filters/http/proto_api_scrubber/scrubbing_util_lib/field_checker_test.cc

```diff
@@ -685,9 +350,13 @@ TEST_F(FieldCheckerTest, RequestFieldChecker) {
     // `envoy.extensions.filters.http.proto_api_scrubber.v3.RemoveFieldAction`
     // and hence, CheckField returns kInclude.
     Protobuf::Field field;
-    field.set_name("filter_criteria.publication_details.original_release_info.region_code");
+    field.set_name("region_code");
     field.set_kind(Protobuf::Field_Kind_TYPE_INT64);
-    EXPECT_EQ(field_checker.CheckField({"id"}, &field), FieldCheckResults::kExclude);
```

Comment:
Any insight on why previously this line had `id` in `field_checker.CheckField({"id"}` and this passed, although the test seemed to be using the `region_code` field?